### PR TITLE
Rename to shipkit part 2

### PR DESCRIPTION
**Pull request by NagRock**
This PR should completely fix https://github.com/mockito/shipkit/issues/153

### CODE REVIEW

File: src/main/groovy/org/shipkit/internal/gradle/util/BuildConventions.java [hunk 1]
```diff
@@ -20,6 +20,6 @@ public static File contributorsFile(Project project) {
      * Returns file object in a standard location where we put other output files generated during the build.
      */
     private static File outputFile(Project project, String fileName) {
-        return new File(project.getBuildDir(), "/release-tools/" + fileName);
+        return new File(project.getBuildDir(), "/mockito/" + fileName);
```
Comment (wwilk) [inline src/main/groovy/org/shipkit/internal/gradle/util/BuildConventions.java]: Why 'mockito' not 'shipkit'?

Comment (NagRock) [inline src/main/groovy/org/shipkit/internal/gradle/util/BuildConventions.java]: Hah! Thanks for review accuracy and catching that.

File: src/main/groovy/org/shipkit/internal/gradle/InitConfigFileTask.java [hunk 2]
```diff
@@ -86,7 +86,7 @@ public void setGitOriginRepoProvider(GitOriginRepoProvider gitOriginRepoProvider
                     "}\n"+
                     "\n"+
                     "allprojects {\n"+
-                    "   plugins.withId(\"org.mockito.mockito-release-tools.bintray\") {\n"+
+                    "   plugins.withId(\"org.mockito.shipkit.bintray\") {\n"+
```
Comment (epeee) [inline src/main/groovy/org/shipkit/internal/gradle/InitConfigFileTask.java]: It should rather be `org.shipkit.bintray`, isn't it?

Comment (NagRock) [inline src/main/groovy/org/shipkit/internal/gradle/InitConfigFileTask.java]: You right, thanks!

**Comment by mstachniuk** [comment]
:+1: for merge at will or continue work in scope in this PR

There are still a lot things to change, ex:
* most of 'mockito' occurrences, specially in files:
build.gradle
settings.gradle
shikit.gradle
Exec
ReleaseConfiguration
BintrayDefaultArtifactUrlResolver
E2ETestingPlugin
ReleaseConfigurationPlugin
TaskMaker
GitUtilTest

* mockito-release-tools-example -> shipkit-example

**Comment by NagRock** [comment]
@mstachniuk most of files you listed, contains `mockito` word because of `mockito` organisation in repo links. For example:
ReleaseConfigurationPlugin contains `configuration.getGitHub().setRepository("mockito/shipkit");`
E2ETestingPlugin contains `e2eTest.create("https://github.com/mockito/shipkit-example");`
ReleaseConfiguration contains `https://github.com/mockito/shipkit/issues/76`
etc.

**Comment by mstachniuk** [comment]
Not exactly:
ReleaseConfigurationPlugin 
* .withProperty("gitHub.repository", "mockito/mockito-release-tools-example") -> .withProperty("gitHub.repository", "mockito/shipkit-example")
E2ETestingPlugin
* e2eTest.create("https://github.com/mockito/mockito-release-tools-example"); -> e2eTest.create("https://github.com/mockito/shipkit-example");
* '$buildDir/project-name-work' and execute 'testRelease' task using the newest mockito-release-tools version ->  '$buildDir/project-name-work' and execute 'testRelease' task using the newest shipkit version
ReleaseConfiguration
 * ttps://github.com/mockito/mockito-release-tools-example/blob/master/gradle/release.gradle -> ttps://github.com/mockito/shipkit-example/blob/master/gradle/release.gradle
 * ttps://github.com/mockito/mockito-release-tools/issues/76" -> ttps://github.com/mockito/shipkit/issues/76"

I know it's long boring process and we can't make global search and replace :-(

You can merge now, or continue work

**Comment by NagRock** [comment]
Sorry, but I think that you have not updated version of code:
https://github.com/mockito/shipkit/pull/214/files#diff-234f0ab26b03e3db0b66b74751b55552L28
https://github.com/mockito/shipkit/pull/214/files#diff-76230f5c264b06d17eda6c82390d3344L18

**Comment by mstachniuk** [comment]
Ok, I was on `rename-to-shipkit` branch instead of `rename-to-shipkit-part-2`. Really sorry for confusion

**Comment by NagRock** [comment]
No problem :)

**Pull request merged**